index.html

```diff
@@ -0,0 +1,115 @@
+                  plant pathology program. Over the next 16 years, his team breeds
+                  6,000 different strains of disease resistent wheat - including
+                  different varieties for each major climate on Earth.
+                </li>
+                <li>
+                  <strong>1945</strong> - Discovers a way to grown wheat twice each
+                  season, doubling wheat yields
+                </li>
+                <li>
+                  <strong>1953</strong> - crosses a short, sturdy dwarf breed of wheat
+                  with a high-yeidling American breed, creating a strain that responds
+                  well to fertilizer. It goes on to provide 95% of Mexico's wheat.
+                </li>
+                <li>
+                  <strong>1962</strong> - Visits Delhi and brings his high-yielding
+                  strains of wheat to the Indian subcontinent in time to help mitigate
+                  mass starvation due to a rapidly expanding population
+                </li>
+                <li><strong>1970</strong> - receives the Nobel Peace Prize</li>
+                <li>
+                  <strong>1983</strong> - helps seven African countries dramatically
+                  increase their maize and sorghum yields
+                </li>
+                <li>
+                  <strong>1984</strong> - becomes a distinguished professor at Texas
+                  A&amp;M University
+                </li>
+                <li>
+                  <strong>2005</strong> - states "we will have to double the world
+                  food supply by 2050." Argues that genetically modified crops are the
+                  only way we can meet the demand, as we run out of arable land. Says
+                  that GM crops are not inherently dangerous because "we've been
+                  genetically modifying plants and animals for a long time. Long
+                  before we called it science, people were selecting the best breeds."
+                </li>
+                <li><strong>2009</strong> - dies at the age of 95.</li>
+              </ul>
+        </div>
+        <blockquote cite="http://news.rediff.com/report/2009/sep/14/pm-pays-tribute-to-father-of-green-revolution-borlaug.htm">
+            <p>
+              "Borlaug's life and achievement are testimony to the far-reaching
+              contribution that one man's towering intellect, persistence and
+              scientific vision can make to human peace and progress."
+            </p>
+            <cite>-- Indian Prime Minister Manmohan Singh</cite>
+          </blockquote>
+       <div id="tribute-link"> <a id="tribute-link" target="_blank"></a></div>
+       <h3>
+        If you have time, you should read more about this incredible human
+        being on his
+        <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">Wikipedia entry</a>.
+      </h3>
+    </div>
+</body>
+</html>
```

Comment:
Please check the ERRORS on linters by clicking details to make sure it passes them.

Reply:
- [ ] Kindly add a  'Why' for every change you request. For the linters you may say something like 'because linters get you in the habit of using best coding practices acknowledged by developers all across the world'.